src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3297,20 +3299,43 @@ protected override IPointerTypeSymbol CommonCreatePointerTypeSymbol(ITypeSymbol
                 throw new ArgumentException(CSharpResources.OutIsNotValidForReturn);
             }
 
+            if (callingConvention != SignatureCallingConvention.Unmanaged && !callingConventionTypes.IsDefaultOrEmpty)
```

Comment:
>callingConventionTypes.IsDefaultOrEmpty [](start = 78, length = 39)

It looks like we disallow passing default arrays to public APIs.

Reply:
I made `callingConventionTypes` optional in the public API, so it can be default.

---
In reply to: [463765979](https://github.com/dotnet/roslyn/pull/46467#discussion_r463765979) [](ancestors = 463765979)